core/cloudflow-operator/src/main/scala/cloudflow/operator/CloudflowLabels.scala

```diff
@@ -21,15 +21,16 @@ case class CloudflowLabels(partOf: String, appVersion: String) {
   import CloudflowLabels._
 
   val baseLabels: Map[String, String] = Map(
-    PartOf    -> partOf,
+    PartOf    -> cloudflow.operator.Name.ofLabelValue(partOf),
     ManagedBy -> CloudflowLabels.ManagedByCloudflow,
-    Version   -> appVersion
+    Version   -> cloudflow.operator.Name.ofLabelValue(appVersion)
```

Comment:
do we need the fully qualified name here ?

Reply:
There's another `Name` in the same file that would result in a conflict.